capi-ruby-units/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN set -ex \
 RUN set -ex \
     && wget https://raw.githubusercontent.com/cloudfoundry/cloud_controller_ng/main/Gemfile.lock \
     && wget https://raw.githubusercontent.com/cloudfoundry/cloud_controller_ng/main/Gemfile \
+    && bundle config set --local without 'development'
```

Comment:
out of curiosity, what happened that this line is now needed?

Reply:
there was an issue, that I also ran into when bundling on our workspaces, where `ruby-debug-ide` wouldn't successfully install on the first try.  Rerunning `bundle install` succeeds for some reason.  But we don't need it for running unit tests, so I just removed it.  There was a similar issue with the `debase` gem in github actions.